.eslintrc

```diff
@@ -1 +1,23 @@
-{}
+{
+    "parser": "@babel/eslint-parser",
+    "extends": ["airbnb"],
+    "rules": {
+        "react/jsx-filename-extension": [
+            1,
+            {
+                "extensions": [".js"]
+            }
+        ],
+        "indent": ["error", 4],
+        "react/jsx-indent": ["error", 4],
+        "react/jsx-indent-props": ["error", 4],
+        "react/state-in-constructor": [0, "never"],
+        "linebreak-style": 0,
+        "ObjectExpression": 0,
```

Comment:
To raczej już przestarzała opcja - do czego jej używałeś?
https://github.com/eslint/eslint/issues/8979

PS. Daj mi znać na Slacku jak odpiszesz :P

Reply:
Takie rozwiązania stackoverflow podpowiadał kiedy szukałem rozwiązań co do niektórych błędów, które fix vsc nie ogarniał, np. ` {fields.map(({ name, type, label }) => {` 
eslint wymagał, aby parametry obiektu zaczynały się od nowej linijki, wówczas nie pokazywał błędu, natomiast kiedy chciałem zapisać zmiany to vsc przywracał powyższe ustawienie i eslnit znów pokazywał błąd :/ tamta opcja rozwiązała ten problem